ED/src/dynamics/disturbance.f90

```diff
@@ -3124,7 +3125,8 @@ subroutine prune_lianas(csite, np, lsl)
                                 , h2dbh                    & ! function
                                 , size2bl                  & ! function
                                 , dbh2bd                   & ! function
-                                , dbh2krdepth              ! ! function
+                                , dbh2krdepth              & ! function
+                                , dbh2sf                   ! ! function
```

Comment:
I see that you added dbh2sf in the "use allometry, only: ..." blocks in several subroutines, but they don't seem to be used.